specification/databricks/resource-manager/Microsoft.Databricks/preview/2021-04-01-preview/databricks.json

```diff
@@ -1532,6 +1578,71 @@
         "name": "PrivateEndpointConnectionProvisioningState",
         "modelAsString": true
       }
+    },
+    "EndpointDependency": {
+      "description": "A domain name or IP address the Workspace is reaching at.",
+      "type": "object",
+      "properties": {
+        "domainName": {
+          "description": "The domain name of the dependency.",
+          "type": "string"
+        },
+        "endpointDetails": {
+          "description": "The Ports used when connecting to domainName.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EndpointDetail"
+          }
+        }
+      }
+    },
+    "EndpointDetail": {
+      "description": "Connect information from the Workspace to a single endpoint.",
+      "type": "object",
+      "properties": {
+        "ipAddress": {
+          "description": "An IP Address that Domain Name currently resolves to.",
+          "type": "string"
+        },
+        "port": {
+          "format": "int32",
+          "description": "The port an endpoint is connected to.",
+          "type": "integer"
+        },
+        "latency": {
+          "format": "double",
+          "description": "The time in milliseconds it takes for the connection to be created from the Workspace to this IpAddress at this Port.",
+          "type": "number"
+        },
+        "isAccessible": {
+          "description": "Whether it is possible to create a connection from the Workspace to this IpAddress at this Port.",
+          "type": "boolean"
+        }
+      }
+    },
+    "OutboundEnvironmentEndpoint": {
+      "description": "Egress endpoints which Workspace connects to for common purposes.",
+      "type": "object",
+      "properties": {
+        "category": {
+          "description": "The category of endpoints accessed by the Workspace, e.g. azure-storage, azure-mysql, etc.",
+          "type": "string"
+        },
+        "endpoints": {
+          "description": "The endpoints that Workspace connect to",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EndpointDependency"
+          }
+        }
+      }
+    },
+    "OutboundEnvironmentEndpointCollection": {
+      "description": "Collection of OutboundEnvironmentEndpoint",
```

Comment:
Change description to  "Collection of outbound network dependency endpoints"

Reply:
Changed description